nextcord/guild.py

```diff
@@ -2496,17 +2491,14 @@ async def create_custom_emoji(
         :class:`Emoji`
             The created emoji.
         """
-
-        img = utils._bytes_to_base64_data(image)
-
-        role_ids: SnowflakeList
```

Comment:
The removal of `role_ids: SnowflakeList` seems unrelated to the PR. This adds a type annotation for the variable.